Testscripts/Linux/TestRDMA_MultiVM.sh

```diff
@@ -715,6 +767,7 @@ function Main() {
 			mpi_run_path=$(find / -name mpirun | grep intel64)
 		;;
 		mvapich)
+			omb_p2p_path="/root/mvapich2-2.3/osu_benchmarks/mpi/pt2pt/"
```

Comment:
If we have a newer version to OSU_MPI_BENCHMARK_DOWNLOAD, then how can you manage this path? Your path has version number.

Reply:
Modified in revision